crates/bifrost/src/providers/replicated_loglet/tasks/trim.rs

```diff
@@ -138,30 +138,36 @@ impl<'a> TrimTask<'a> {
             });
         }
 
+        // If we get a quorum, we will acknowledge the trim request with the highest trim point
+        // reported by any log server. This may be higher than what was requested.
+        let mut response_trim_point = LogletOffset::INVALID;
```

Comment:
On re-reading the change, I realized that the replicated loglet retured as the actual trim point that reported by the latest log server to respond as we reach the write consensus requirement. A more paranoid approach is to take the max of all the reported trim points thus far - this could be higher than was was requested, but it's probably safer to over-report.